tools/seahorn/seahorn.cpp

```diff
@@ -312,6 +312,11 @@ int main(int argc, char **argv) {
 
   // -- initialize any global variables that are left
   pass_manager.add(new seahorn::LowerGvInitializers());
+
+  // TODO: get rid of the deprecaed dsa pass??
+
+  // the ShadowMemSeaDsa pass is now added automatically because it is required
+  // by HornifyModule
```

Comment:
Not sure what this comment means. But note that `HornifyModule` is not always loaded -- for example when we do Bmc. While Dsa and Shadow are still required.

Reply:
This comment is outdated, I forgot to remove it.